ansible/templates/usb/scripts/computenode.sh.j2

```diff
@@ -229,8 +230,8 @@ function reload_smf_manifests() {
 	printf_log "\r%-64s" "Reloading SMF manifests... "
 	printf_log "%4s\n" "done"
 	# This will make the output to look nicer/aligned
-	/usr/sbin/svcadm disable -s svc:/system/manifest-import:default
-	/usr/sbin/svcadm enable -s svc:/system/manifest-import:default
+	svcadm disable -s svc:/system/manifest-import:default
+	svcadm enable -s svc:/system/manifest-import:default
```

Comment:
It depends... In this case, we have defined a PATH, so it is better to use relative paths.